corehq/apps/geospatial/tasks.py

```diff
@@ -0,0 +1,24 @@
+from celery import states
+from celery.result import AsyncResult
+
+from corehq.apps.celery import task
+from corehq.apps.geospatial.utils import update_cases_owner
+
+
+@task(queue="background_queue", ignore_result=False)
+def geo_cases_reassignment_update_owners(domain, case_id_to_owner_id):
+    update_cases_owner(domain, case_id_to_owner_id)
+
+
+def is_task_invoked_and_not_completed(task_id):
```

Comment:
Nit: what do you think of the following function name: `task_is_busy` ?

Reply:
Also, another way of doing this is to set a key in redis. Its commonly done in HQ and has its own advantages.
I feel with celery this could get error prone.

Reply:
The approach has been modified to use redis and is available in commits e724ddb914d37a0afd8d533ad6014ef35764e6de and 6b2cb2a1e8fdba1186ffcd4532cc987ce4a21ffd